ground/gcs/src/libs/tlmapcontrol/mapwidget/tlmapwidget.h

```diff
@@ -45,6 +45,7 @@
 #include "mapcircle.h"
 #include "waypointcurve.h"
 #include "waypointitem.h"
+#include "../core/corecommon.h"
```

Comment:
I don't like this-- perhaps a better place can be found for this header?  (I'd prefer to not use relative include paths like this that presume a certain repository layout, and it doesn't really seem to be specific to the core plugin).